CRM/Contact/Form/Edit/CommunicationPreferences.php

```diff
@@ -84,7 +84,7 @@ public static function buildQuickForm(&$form) {
     //using for display purpose.
     $form->assign('commPreference', $commPreference);
 
-    $form->addField('preferred_mail_format', array('entity' => 'contact', 'label' => ts('Email Format')));
+    $form->addField('preferred_mail_format', array('entity' => 'contact', 'label' => ts('Email Format')), TRUE);
```

Comment:
This throws a validation error when attempting to create a new contact from "New Individual" block.